spec/lang/step/operators.md

```diff
@@ -278,15 +279,17 @@ impl<M: Memory> Machine<M> {
         (left, l_ty): (Value<M>, Type),
         (right, _r_ty): (Value<M>, Type)
     ) -> Result<(Value<M>, Type)> {
-        let Value::Ptr(left) = left else { panic!("non-pointer left input to `PtrOffset`") };
+        let Value::Ptr(Pointer { thin_pointer: left, .. }) = left else {
+            panic!("non-pointer left input to `PtrOffsetFrom`")
```

Comment:
You changed the error message here...

Also please make this panic if there is metadata.